common/src/chain/config.rs

```diff
@@ -117,6 +116,21 @@ pub fn create_mainnet() -> ChainConfig {
     }
 }
 
+pub fn unit_test_config() -> ChainConfig {
```

Comment:
`create_unit_test_config()`. Let's follow the same convention.

Reply:
Done.